modules/150-user-authn/images/dex-authenticator/werf.inc.yaml

```diff
@@ -0,0 +1,95 @@
+---
+image: {{ .ModuleName }}/{{ .ImageName }}
+fromImage: common/distroless
+import:
+  - image: {{ $.ModuleName }}/{{ $.ImageName }}-oauth2-proxy-artifact
+    add: /src/oauth2-proxy
+    to: /usr/local/bin/oauth2_proxy
+    before: setup
+  - image: {{ $.ModuleName }}/{{ $.ImageName }}-url-exec-prober-artifact
+    add: /src/url-exec-prober
+    to: /usr/local/bin/url-exec-prober
+    before: setup
+docker:
+  ENTRYPOINT: [ "/bin/oauth2_proxy" ]
+  CMD: [ "--upstream=http://0.0.0.0:8080/", "--http-address=0.0.0.0:4180" ]
+---
+image: {{ .ModuleName }}/{{ .ImageName }}-oauth2-proxy-src-artifact
+fromImage: common/src-artifact
+final: false
+git:
+  - add: /{{ $.ModulePath }}modules/{{ $.ModulePriority }}-{{ $.ModuleName }}/images/{{ $.ImageName }}/patches/
+    to: /patches
+  - url: {{ $.SOURCE_REPO }}/oauth2-proxy/oauth2-proxy.git
+    to: /src
+    tag: v7.5.1
+    stageDependencies:
+      install:
+        - '**/*.go'
+        - '**/go.mod'
+        - '**/go.sum'
+shell:
+  install:
+    - cd /src
+    - git apply -v /patches/cookie-refresh.patch
+    - git apply -v /patches/remove-groups.patch
+    - git apply -v /patches/deps.patch
```

Comment:
if we add numbers to patch names we can do it by one command - `git apply --verbose /patches/*.patch`

Reply:
done